trigger/src/main/scala/core/Test.scala

```diff
@@ -0,0 +1,86 @@
+package core
+
+import core.models.tables._
+import core.persistance._
+import core.scheduler.JobScheduler
+import core.scheduler.eventProcessor.EventProcessor
+import core.scheduler.executors.Executors
+import core.scheduler.executors.spark.App
+import core.scheduler.sensors.Sensors
+import play.api.libs.json.{Json, OFormat}
+import slick.dbio.DBIO
+import slick.jdbc.MySQLProfile.api._
+
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration
+
+object CreateTables {
+  def main(args: Array[String]): Unit = {
+    val db = Database.forConfig("mysqlDB")
```

Comment:
Is this class expected to be a test for MySQL, in case it is, maybe it would be better to put it in the `test` source tree.

Reply:
I have accidentally committed this class. Will be removed.